content/en/blog/2023/2023-10-15-kcs-shanghai/index.md

```diff
@@ -0,0 +1,107 @@
+
+![Group Photo](./images/kcs06.jpeg)
+
+The attending contributors were no longer just GitHub IDs; they transformed into vivid faces.
+From sitting together and capturing group photos to attempting to identify "Who is who,"
+a loosely connected collective emerged. This team structure, although loosely knit and free-spirited,
+was established to pursue shared dreams.
+
+As the saying goes, "You reap what you sow." Each effort has been diligently documented within
+the Kubernetes community contributions. Regardless of the passage of time, the community will
+not erase those shining traces. Brilliance can be found in your PRs, issues, or comments.
+It can also be seen in the smiling faces captured in meetup photos or heard through stories passed down among contributors.
+
+## Technical sharing and discussions
+
+Next, there were three technical sharing sessions:
+
+- [sig-multi-cluster](https://github.com/kubernetes/community/blob/master/sig-multicluster/README.md):
+  [Hongcai Ren](https://github.com/RainbowMango), the owner of Karmada, provided an introduction to
+  the responsibilities and roles of this SIG. Their focus is on designing, discussing, implementing,
+  and maintaining APIs, tools, and documentation related to multi-cluster management.
+  Cluster Federation, one of Karmada's core concepts, is also part of their work.
+
+- [helmfile](https://github.com/helmfile/helmfile): [yxxhero](https://github.com/yxxhero)
+  from [GitLab](https://gitlab.cn/) presented how to deploy Kubernetes manifests declaratively,
+  customize configurations, and leverage the latest features of Helm, including Helmfile.
+
+- [sig-scheduling](https://github.com/kubernetes/community/blob/master/sig-scheduling/README.md):
+  [william-wang](https://github.com/william-wang) from Huawei Cloud shared the recent updates and
+  future plans of SIG Scheduling. This SIG is responsible for designing, developing, and testing
+  components related to Pod scheduling.
+
+![Technical Session](./images/kcs03.jpeg)
+
+Following the sessions, a video featuring a call for contributors by [Sergey Kanzhelev](https://github.com/SergeyKanzhelev),
+the SIG-Node Chair, was played. The purpose was to encourage more contributors to join the Kubernetes community,
+with a special emphasis on the popular SIG-Node.
+
+Lastly, Kevin hosted an Unconference collective discussion session covering topics such as
+multi-cluster management, scheduling, elasticity, AI, and more. For detailed minutes of
+the Unconference meeting, please refer to <https://docs.qq.com/doc/DY3pLWklzQkhjWHNT>.
+
+## China's contributor statistics
+
+The contributor summit took place in Shanghai, with 90% of the attendees being Chinese.
+Within the Cloud Native Computing Foundation (CNCF) ecosystem, contributions from China have been steadily increasing. Currently:
+
+- Chinese contributors account for 9% of the total.
+- Contributions from China make up 11.7% of the overall volume.
```

Comment:
This probably understates Chinese contributions.  A lot of Chinese contributors use VPNs and may not show up as being from China in the stats.  I don't know a way to get better figures, though.

Reply:
Thanks for you tips. Add a note there. Hope it is helpful.